src/usr/local/www/diag_ping_shared.php

```diff
@@ -0,0 +1,76 @@
+<?php
+/*
+ * diag_ping.php
+ *
+ * part of pfSense (https://www.pfsense.org)
+ * Copyright (c) 2004-2017 Rubicon Communications, LLC (Netgate)
+ * Copyright (c) 2003-2005 Bob Zoller (bob@kludgebox.com)
```

Comment:
This file only contain bootstrap forms, I don't believe there are any content added by Bob here